src/Components/ContactUserCard.js

```diff
@@ -0,0 +1,38 @@
+const ContactUserCard = [
+    {
+        Name: "Timmy",
+        Image: "src='https://www.google.com/imgres?imgurl=https%3A%2F%2Fstatic.wikia.nocookie.net%2Fsouthpark%2Fimages%2F4%2F4b%2FTimmy.png%2Frevision%2Flatest%3Fcb%3D20171014061716&imgrefurl=https%3A%2F%2Fsouthpark.fandom.com%2Fwiki%2FTimmy_Burch&tbnid=q0ohnO6HHfV4pM&vet=12ahUKEwjCzq-o9fP8AhU8lXIEHYeIDGUQMygRegUIARCTAQ..i&docid=lXEgFnNMUVWWfM&w=524&h=1029&q=timmy&ved=2ahUKEwjCzq-o9fP8AhU8lXIEHYeIDGUQMygRegUIARCTAQ'",
+        Title: "Student",
+        Description: "A student attending South Park Elementary School"
+    },
+    {
+        Name: "Courage",
+        Image: "src='https://www.google.com/url?sa=i&url=https%3A%2F%2Fcourage.fandom.com%2Fwiki%2FCourage&psig=AOvVaw3t_r3rR1CuhpLSRrCpB0Be&ust=1675327106769000&source=images&cd=vfe&ved=0CA8QjRxqFwoTCJju8Pv18_wCFQAAAAAdAAAAABAE'",
+        Title: "Dog",
+        Description: "An adventurous dog that goes on many perilous adventures in order to ensure the safety of his caretakers"
+    },
+    {
+        Name: "Example 3",
+        Image: "none",
+        Title: "Example Title",
+        Description: "Example Description",
+    },
+    {
+        Name: "Example 4",
+        Image: "none",
+        Title: "Example Title",
+        Description: "Example Description"
+    }
+]
+
+/*
+Contact Example:
+[{
+    Name: "MyName",
+    Image: "",
+    Title: "",
+    Description: ""
+}]
+*/
+
+export default ContactUserCard
```

Comment:
ContactUserCard should be a react component (i.e. a function that returns a JSX expression), not a JavaScript array.